cdap-app-templates/cdap-etl/cdap-etl-batch/pom.xml

```diff
@@ -66,6 +66,17 @@
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.avro</groupId>
+      <artifactId>avro</artifactId>
+      <version>${apache.avro.version}</version>
```

Comment:
Why specify version here? We should be using dependency management in the parent pom.